package/gluon-mesh-babel/files/etc/hotplug.d/iface/90-gluon-babel-brclient-no-igmp-snooping

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+case "$ACTION"
+in
+	ifup)
+	echo 1 > /sys/devices/virtual/net/br-client/bridge/multicast_snooping
```

Comment:
The latest netifd version in LEDE has disabled multicast_snooping for the LEDE release because it causes too many issues. I think we should do the same in Gluon for now... :/